code/game/objects/items/stunbaton.dm

```diff
@@ -26,6 +29,9 @@
 	user.visible_message("<span class='suicide'>[user] is putting the live [name] in [user.p_their()] mouth! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	return (FIRELOSS)
 
+/obj/item/melee/baton/proc/get_wait_description()

```

Comment:
this should be a variable, i'm not seeing why it's malleable anyways since classicbaton is typed differently.